.eslintrc

```diff
@@ -45,6 +45,21 @@
 			{
 				"required": "id"
 			}
-		]
+		],
+		"inclusive-language/use-inclusive-words": [
+			"warn",
```

Comment:
If there are no more violations to this rule, can we just set it to `error`? I'm personally not a fan of lint warnings, either something is wrong or it doesn't matter.

Reply:
@DanReyLop: yep we can, I'll make adjustments.